rush-plugins/rush-amazon-s3-build-cache-plugin/src/integration-tests/readObject.ts

```diff
@@ -0,0 +1,30 @@
+import { AmazonS3Client } from '../AmazonS3Client';
```

Comment:
Stick this in its own project under the `build-tests` folder so we don't have to use `ts-node`

Reply:
👍